torch/onnx/symbolic_opset9.py

```diff
@@ -305,6 +305,15 @@ def expand(g, self, size, implicit):
     size = sym_help._maybe_get_const(size, 'is')
     if not sym_help._is_value(size):
         size = g.op("Constant", value_t=torch.LongTensor(size))
+    else:
+        # Expand with -1 dim value means dim is unchanged.
+        # Since onnx::expand supports two-way broadcasting,
+        # -1 dim value can be exported to onnx as 1
+        list_ = stack(g, size, 0)
```

Comment:
Need to check if `size` is list before applying stack. it seems `size` could be a tensor.